### PR TITLE
Austa 266, issue #51, Step "Deploy Win10 DVD ISO on Windows Worker" has Error "deployRoot doesn't start with /"

### LINKED ISSUE #51: Step "Deploy Win10 DVD ISO on Windows Worker" has Error "deployRoot doesn't start with /"
Step `Deploy Win10 DVD ISO on Windows Worker` has the error:
```
deployRoot doesn't start with /
```
image

### PR DESCRIPTION
**Pull request by h4xhor**
Modified step `Deploy Win10 DVD ISO on Windows Worker` and added step `Move Windows DVD ISO to Build Directory`.

closes https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/issues/51 and https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/issues/55.

### CODE REVIEW

File: steps/movewindowsdvdisotobuilddirectory/script.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+$CURRENT_PATH = "C:\software\build-{newOsNode.fqn}"
+
+Set-Location "${CURRENT_PATH}"
+
+$ISO = (dir *.ISO).name
+
+$DEST_FOLDER = "{automationWorkerWindowsBaseDirectory}\build-{newOsNode.fqn}"
```
Comment (brentonford) [inline steps/movewindowsdvdisotobuilddirectory/script.txt]: @h4xhor, are you removing the automationWorkerWindowsBaseDirectory parameter?

**Review by brentonford — CHANGES_REQUESTED** [review]
@h4xhor, are you replacing the parameter (automationWorkerWindowsBaseDirectory) with a file path?

File: steps/movewindowsdvdisotobuilddirectory/script.txt (hunk 1 above)
Comment (h4xhor) [reply steps/movewindowsdvdisotobuilddirectory/script.txt]: Hi @brentonford , no. I am still using it.

Comment (h4xhor) [reply steps/movewindowsdvdisotobuilddirectory/script.txt]: Hi @brentonford , I've removed parameter `automationWorkerWindowsBaseDirectory` and hard coded it as `C:/attuneautomationworker`.

**Pull request merged**